pandas/_libs/tslibs/fields.pyx

```diff
@@ -325,7 +325,8 @@ def get_start_end_field(
 
 @cython.wraparound(False)
 @cython.boundscheck(False)
-def get_date_field(const int64_t[:] dtindex, str field, NPY_DATETIMEUNIT reso=NPY_FR_ns):
+def get_date_field(const int64_t[:] dtindex, str field,. 
```

Comment:
looks like you've added a `.` at the end of the line

Reply:
i'll fix it